dashboard/app/views/pd/regional_partner_contact_mailer/unmatched.html.haml

```diff
@@ -1,6 +1,6 @@
 %p
   The following person has requested to connect with their local Code.org Regional Partner
-  = link_to('through this form,', 'https://code.org/educate/professional-learning/contact-regional-partner')
+  = link_to('through this form,', CDO.code_org_url('educate/professional-learning/contact-regional-partner', CDO.default_scheme))
```

Comment:
The scheme should be optional; is it needed here?

Reply:
TBH I followed precedent from the other mailer -- I looked up what this default_scheme variable contains (http vs https), but couldn't find anything in Rails docs about how it is used in a `link_to`

Reply:
Oh, because I misinterpreted this as an argument to link_to, which it is not 🤦‍♂ No wonder I couldn't find any docs!

Reply:
Ah, sounds like a protocol relative URL is now [out of fashion](https://stackoverflow.com/questions/16869320/is-it-safe-to-use-protocol-relative-url-in-email) though it's quite useful for our own development environments.  

But it sounds like there are [particular pitfalls](http://billpatrianakos.me/blog/2013/04/18/protocol-relative-urls/) when they're used in emails, so that's probably a good reason to specify the protocol explicitly here.